apps/dashboard/lib/smart_attributes/attribute.rb

```diff
@@ -21,10 +21,15 @@ def fixed?
       !!opts[:fixed]
     end
 
-    # Value of attribute
+    def hide_when_empty?
+      !!opts[:hide_when_empty]
+    end
+
+    # Value of attribute. It is converted to String.
+    # To support HTML file inputs, if the attribute is detected as as file input, it is not converted.
```

Comment:
The behaviour of the session information in the form is driven by its presence.
When a session was selected, show the information in the form and send the id to the bac- end. When no session is selected, the field should not appear in the form. Is sort of the opposite of the `fixed` property

Reply:
I'm having a think on this. I see how you arrived at it/why we need it, I just wonder if there's another way. This is likely the simplest...

Reply:
This will just have to do.